app/scripts/views/complete_reset_password.js

```diff
@@ -61,10 +52,13 @@ function (Cocktail, BaseView, FormView, Template, PasswordMixin,
       }
 
       var self = this;
-      return this.fxaClient.isPasswordResetComplete(this.token)
+      var verificationInfo = this._verificationInfo;
+      var token = verificationInfo.get('token');
+      return this.fxaClient.isPasswordResetComplete(token)
         .then(function (isComplete) {
-          self._isLinkExpired = isComplete;
           if (isComplete) {
+            verificationInfo.setValidationError(
+                AuthErrors.toError('EXPIRED_VERIFICATION_LINK'));
```

Comment:
This feels like a "separate feature", i.e showing that link has expired to the user. 
Does it make sense to split the commits or change the commit from:
`refactor(client): Extract email verification info into their own models.`
to `fix(client): Add expired email error and extract email verification info into their own models`

The goal here is to make sure this change gets listed in the changelog so no one is surprised that we added this. We had this issue in the past that the changelog was missing a `refactor` commit that had major changes to metrics logging